.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,21 @@
+name: golangci-lint
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
```

Comment:
Turns out these lines are a bad idea. See gonum#1330.

Reply:
ah, indeed, what I had in go-hep was:
```
on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
  schedule:
    - cron: '0 2 * * 1-5'
```